hazelcast/src/main/java/com/hazelcast/monitor/LocalScheduledExecutorStats.java

```diff
@@ -0,0 +1,5 @@
+package com.hazelcast.monitor;
+
+
+public interface LocalScheduledExecutorStats extends LocalInstanceStats {
```

Comment:
'Local' is misleading. 
Are these stats per scheduler, per node? The current implementation is per scheduler.

Reply:
This class needs to be moved to implementation since it can't be accessed from the outside anyway.

Reply:
And local.. that is is how all these statistics classes are called.